workspace-manager-clienttests/src/main/resources/measurementlists/DemoList.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Let's remove this file for now. This file is used for the collectMeasurement Gradle task, which is not tested outside Data Repo yet. I think this first PR should focus just on running tests (runTest Gradle task)

Reply:
Removed DemoList.java